### PR TITLE
Add cards for single vlqFH in Run3

### PR DESCRIPTION
**Pull request by sarakm0704**
Adding cards for single-produced VLQ (TprimeBToTH_M-650_LH_TuneCP5_PSweights_14TeV-madgraph_pythia8) in Run3

previously Run2 sample was produced with:
[bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tH_b/tH_b_LH_narrow_M600](https://github.com/cms-sw/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tH_b/tH_b_LH_narrow_M600)

Only the beam energy has changed (13TeV -> 13p6TeV)

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/13p6TeV/tH_b_LH_narrow_M600/tH_b_LH_narrow_M600_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,17 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/13p6TeV/tH_b_LH_narrow_M600/tH_b_LH_narrow_M600_proc_card.dat:9]: these lines are not needed

File: bin/MadGraph5_aMCatNLO/cards/13p6TeV/tH_b_LH_narrow_M600/tH_b_LH_narrow_M600_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+     6800     = ebeam2  ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+# 'cteq6l1'    = pdlabel     ! PDF set (lhapdf for using lhapdf)
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/13p6TeV/tH_b_LH_narrow_M600/tH_b_LH_narrow_M600_run_card.dat:51]: remove line 51

**Review by menglu21 — COMMENTED** [review]
Hi @sarakm0704 besides the comments inline, please remove those cards to MadGraph5_aMCatNLO/cards/production/13p6TeV/

**Comment by sarakm0704** [comment]
Hi @menglu21, thank you for the comments. I modified the lines and moved in cards/production.

**Pull request merged**